src/dev-theme.css

```diff
@@ -220,6 +220,40 @@ ul > li {
   background: var(--rgba-primary-3);
 }
 
+/*Footer*/
+.themed-footer-header{
+  font-family: 'Noto Serif';
+  font-size: calc(1.5em + 1vw);
+  padding-top:1rem;
+}
+.themed-footer-header[data-theme='light']{
+  color: var(--rgba-secondary-2-3);
+}
+
```

Comment:
Could you create a new .themed-footer class without the data-theme selector and move the padding and margin calls into it?